hardware_pydantic/lab_objects.py

```diff
@@ -112,6 +239,24 @@ def move(containee: LabContainee, dest_container: LabContainer, lab: Lab, dest_s
 
     @staticmethod
     def get_container(containee: LabContainee, lab: Lab, upto: Type = None) -> LabContainer | None:
+        """
+        Get the container that contains the containee.
+
+        Parameters
+        ----------
+        containee : LabContainee
+            The containee to get the container from.
+        lab : Lab
+            The lab that contains the containee.
+        upto : Type
```

Comment:
This is to limit the container's "level" in recursion, `None` means the highest level